Covid-19 _Real-time_Notification/Covid.py

```diff
@@ -0,0 +1,39 @@
+from plyer import notification
+import requests
+from bs4 import BeautifulSoup
+import time
+
+def notify_user(title, message):
+    notification.notify(
+    title = title,
+    message = message,
+    app_icon = "./Covid-19_Real-time_Notification/Notify_icon.ico" ,
+    timeout = 3)
+    
+def getInfo(url):
+    r = requests.get(url)
+    return r.text
+
+
+if __name__ == '__main__':
+        states = list(map(str,input("Enter the name of states:").split(",")))
+        
+        while True:
+            HtmlData = getInfo('https://www.medtalks.in/live-corona-counter-india')
+            soup = BeautifulSoup(HtmlData, 'html.parser')
+        
+            myData = ""
+            for tr in soup.find('tbody').find_all('tr'):
+                myData += tr.get_text()
+            myData = myData[1:]
+            itemList = myData.split("\n\n")
+            for item in itemList[:-2]:
+                dataList = item.split('\n')
+            
```

Comment:
if you print the dataList, you will see that the states data is not cleaned:

![image](https://user-images.githubusercontent.com/43691873/116347197-635ad000-a809-11eb-94c3-b9ac57e495c6.png)

This causes error in the script. The script doesn't notify anything as no-one will write `Chandigarh ( चण्डीगढ़ )`. A user will enter `Chandigarh` and that's why no notification was popped up.

Please fix all the suggested issues, run it on your local system and then push your changes